src/py42/services/orgs.py

```diff
@@ -95,7 +93,7 @@ def get_all(self, **kwargs):
     def block(self, org_id):
         """Blocks the organization with the given org ID as well as its child organizations. A
         blocked organization will not allow any of its users or devices to log in. New
-        registrations will be rejected and all currently logged in clients will be logged out.
+        registrations will be rejected and all currently logged in services will be logged out.
```

Comment:
lol I think this one should _actually_ be `clients`.

Reply:
lol I tried to be careful about this, good catch